src/crab/ebpf_domain.cpp

```diff
@@ -607,13 +586,11 @@ static std::vector<linear_constraint_t> assume_unsigned_64bit_gt(const NumAbsDom
     }
 }
 
-static std::vector<linear_constraint_t>assume_unsigned_32bit_gt(const NumAbsDomain& inv, bool strict,
-                                                                variable_t left_svalue, variable_t left_uvalue,
-                                                                const crab::interval_t& left_interval_low,
-                                                                const crab::interval_t& left_interval_high,
-                                                                const linear_expression_t& right_svalue,
-                                                                const linear_expression_t& right_uvalue,
-                                                                const crab::interval_t& right_interval) {
+static std::vector<linear_constraint_t>
+assume_unsigned_32bit_gt(const NumAbsDomain& inv, bool strict, variable_t left_svalue, variable_t left_uvalue,
```

Comment:
Why the line break after the type in this line and others above?  This looks inconsistent with 611, 694, etc.

Reply:
No idea.
```suggestion
static std::vector<linear_constraint_t> assume_unsigned_32bit_gt(const NumAbsDomain& inv, bool strict, variable_t left_svalue, variable_t left_uvalue,
```